src/Cake/Program.cs

```diff
@@ -76,6 +76,18 @@ private static IContainer CreateContainer(bool mono)
             builder.RegisterType<WindowsRegistry>().As<IRegistry>().SingleInstance();
             builder.RegisterType<CakeContext>().As<ICakeContext>().SingleInstance();
 
+            // NuGet addins support
+            builder.RegisterType<NuGetVersionUtilityAdapter>()
+                .As<INuGetFrameworkCompatibilityFilter>()
+                .As<IFrameworkNameParser>()
+                .SingleInstance();
+            builder.RegisterType<NuGetPackageAssembliesLocator>().As<INuGetPackageAssembliesLocator>().SingleInstance();
+            builder.RegisterType<PackageReferenceBundler>().As<IPackageReferenceBundler>().SingleInstance();
+            builder.RegisterType<NuGetAssemblyCompatibilityFilter>()
+                .As<INuGetAssemblyCompatibilityFilter>()
```

Comment:
What's your preferred max line width?  I want to make sure to get put that in my ReSharper settings (ReSharper is what caused the line breaks).